website/docs/add-secure-apps/providers/ssf/index.md

```diff
@@ -36,12 +45,19 @@ Note that authentik doesn't specify which events to subscribe to; instead the ap
 
 ### authentik (the transmitter)
 
-To configure authentik as a shared signals transmitter, the authentik Admin [creates a new provider](./create-ssf-provider), selecting the type "SSF", to serve as the backchannelprovider for the application.
+To configure authentik as a shared signals transmitter, navigate to your authentik instance's Admin interface. [Create a new provider](./create-ssf-provider), selecting the type "SSF", to serve as the backchannel provider for the application.
 
-When creating the SSF provider you will need to select a signing key. This is the key that the Security Event Tokens (SET) is signed with.
+When creating the SSF provider, you will need to select a key to sign Security Event Tokens (<abbr>SET</abbr>). This key is used to sign the SETs that are sent to the application. The application will use this key to verify the SETs.
 
-Optionally, you can specify a event retention time period: this value determines how long events are stored for. If an event could not be sent correctly, and retries occur, the event's expiration is also increased by this duration.
+You may also specify a event retention time period. If an event could not be sent correctly, and retries occur, the event's expiration is also increased by this duration.
+
+:::info{title="Events and SETs"}
+
+Be aware that events created via Security Event Tokens differ from than administrative events displayed in authentik's admin interface under **Events**.
 
-:::info
-Be aware that the SET events are different events than those displayed in the authentik Admin interface under **Events**.
 :::
+
+## External references
```

Comment:
We have a link (or I did) to the spec up above, in the first paragraph. I'd like to leave it there, where we describe what SSF is. Best practice is to link in-line, exactly where the reader will wants the info, and exactly where we reference it.

I'm not ready to commit to adding this new section to new docs, nor all of the docs (we need to be consistent)... nor to the work of maintaining them. If we were 10-person Docs team, maybe, but we need to keep it lean and mean and easy to maintain going forward.